Source/Dafny/Resolver.cs

```diff
@@ -16980,10 +16980,11 @@ class MethodCallInformation {
       bindings.AcceptArgumentExpressionsAsExactParameterList(actuals);
     }
 
-    private static string GetLocationInformation(Formal formal, int formalsCount, int index) {
+    private static string GetLocationInformation(List<Formal> formals, int index) {
+      var formal = formals[index];
       var displayName = formal.HasName && !(formal is ImplicitFormal);
       var description = "";
-      if (formalsCount > 1) {
+      if (formals.Count() > 1) {
```

Comment:
No need to change anything, but I prefer `.Length` over `.Count()` when it is available, to indicate that it's constant time.